src/test/java/TaskListTest.java

```diff
@@ -0,0 +1,34 @@
+import duke.Task;
+import duke.TaskList;
+import duke.Todo;
+import org.junit.jupiter.api.Test;
+
+import java.util.ArrayList;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class TaskListTest {
+    @Test
+    public void markCompleteTest() {
+        ArrayList<Task> tasklist = new ArrayList<>();
+        tasklist.add(new Todo("test 1"));
+        tasklist.add(new Todo("test 2"));
+        tasklist.add(new Todo("test 3"));
+        TaskList tasks = new TaskList(tasklist, 3);
+        tasks.markComplete("done 2");
+        String finished = String.valueOf(tasklist.get(1));
+        assertEquals("[T][✓] test 2", finished);
```

Comment:
Nice junit tests! Perhaps it would be better if the function name followed the junit convention which is name_name_name (i think).